snippets/ossm-current-version-support-snippet.adoc

```diff
@@ -4,4 +4,4 @@
 
 :_mod-docs-content-type: SNIPPET
 
-The most current version of the {SMProductName} Operator can be used with all supported versions of {SMProductShortName}. The version of {SMProductShortName} is specified using the `ServiceMeshControlPlane`.
+The most current version of the {SMProductName} Operator can be used with all supported versions of {SMProductShortName}. The version of {SMProductShortName} is specified using the `ServiceMeshControlPlane` resource.
```

Comment:
```suggestion
The most current version of the {SMProductName} Operator can be used with all supported versions of {SMProductShortName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource.
```
I realize this suggestion is out of the scope of your changes, but it's one little change that would make the text align with our style guidelines. 
![Screenshot 2024-08-15 at 3 42 34 PM](https://github.com/user-attachments/assets/fa542243-7a73-4250-a7c9-cc77c72bfd40)

Reply:
Good catch! Thank you.